modular_skyrat/modules/gun_cargo/code/objects/guns/izhevsk.dm

```diff
@@ -23,7 +23,8 @@
 	alt_icons = TRUE
 	dirt_modifier = 1
 	company_flag = COMPANY_IZHEVSK
-	spread = 29
+	spread = 18
```

Comment:
nah the ak's shit on purpose, leave it

Reply:
> nah the ak's shit on purpose, leave it

It's FAR too bad, it usually harms the civilians isntead of the targer, isntead we are giving the AK recoil.